src/api/track.ts

```diff
@@ -9,10 +9,10 @@ import Storage from '../storage';
 
 import TripsAPI from './trips';
 
-import type { RadarTrackParams, RadarTrackResponse } from '../types';
+import type { RadarTrackParams, RadarTrackResponse, RadarTrackTokenResponse } from '../types';
 
 class TrackAPI {
-  static async trackOnce(params: RadarTrackParams) {
+  static async trackOnce(params: RadarTrackParams, verified: boolean = false, encrypted: boolean = false, host: string | undefined = undefined) {
```

Comment:
Lets just roll these into `RadarTrackParams`

Reply:
Or add another object that is like `verifiedObjects` or something
```ts
static async trackOnce(params: RadarTrackParams, verifiedOptions?: RadarVerifiedOptions) {
```

Reply:
I thought about adding it to `RadarTrackParams`, but we don't want to allow developers to pass it explicitly. More an internal implementation detail. Whether we pass `encrypted = true` (which we will prob rename or remove, btw) is implicit in whether the developer is calling `trackVerified()` or `trackVerifiedToken()

Reply:
We should still wrap it in an optional object. Otherwise it's hard to tell what the arguments are at the callsite:
```ts
TrackAPI.trackOnce(params, true, false, 'https://api-verified.radar.io');
```
As opposed to:
```ts
TrackAPI.trackOnce(params, {
  verified: true, 
  encrypted: false, 
  host: 'https://api-verified.radar.io',
});
```